.github/workflows/maven.yml

```diff
@@ -32,19 +32,19 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        java: [11]
+        java: [17]
       fail-fast: false
 
     runs-on: ${{ matrix.os }}
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
       - name: Set up JDK
-        uses: actions/setup-java@v3
+        uses: actions/setup-java@v4
         with:
-          java-version: 11
+          java-version: 17
```

Comment:
Perhaps `matrix.java` could be used instead.

Reply:
then just do it ;-)

Reply:
you are already changing the area, improve for the sake of DRY

Reply:
GH has Suggest a change feature which I can just accept.